### DISCUSSION TITLE
请问有没有办法对汉语拼音的完整识别

### DISCUSSION THREAD

**Discussion by running-az** [PaddlePaddle/PaddleOCR#13865] [Q&A]
![result_010](https://github.com/user-attachments/assets/b5453477-4988-4867-b9d8-ef67dc825357)
![image](https://github.com/user-attachments/assets/1d7fc28c-6430-454c-be7f-5cba72703bf6)
 文本检测模型和文本识别模型分别用的是：ch_PP-OCRv4_det_infer  、  ch_PP-OCRv4_rec_infer  ，但是识别不出拼音的音调，需要重新训练吗？有无类似的数据集可以提供呢？

**Comment by kerry-weic** [comment]
现有字典中，没有这类音调的字符，需重新训练

**Reply by running-az** [reply]
训练的数据集是只用带音调的拼音就可以，还是需要类似以上图片整个古诗的截图，再标注？？数据集训练可以在gpu版本训练，cpu版本使用吗？

**Reply by UserWangZz** [reply]
带音调的拼音图片即可，并且需要将对应的字符添加到字典中

**Reply by running-az** [reply]
在服务器训练有官方指导书吗？

**Reply by UserWangZz** [reply]
文档里有微调指南

**Reply by running-az** [reply]
可以用服务器训练吗？也是类似的方法吗？

**Reply by GreatV** [reply]
@running-az 先阅读官方文档吧

**Reply by running-az** [reply]
最近一直有在看，在cpu上训练过，但是花费时间太长了，想到是否可以在服务器上训练加快训练速度；然后在电脑上使用训练后的模型

**Reply by running-az** [reply]
26个带声调的英文字母（26x4）就行，还是要类似：kōng 这样带声调的汉字的完整汉语拼音？

**Reply by UserWangZz** [reply]
字典只需要单个字符，确保每个拼音里面的字符都在字典里单独出现过即可
